### PR TITLE
plan, expression: add test and remove useless code

### PR DESCRIPTION
**Pull request by lamxTyler**
@shenli @coocood @zimulala @hanfei1991 @winoros PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: expression/constant_test.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/ast"
+	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/mysql"
+	"github.com/pingcap/tidb/util/mock"
+	"github.com/pingcap/tidb/util/testleak"
+	"github.com/pingcap/tidb/util/types"
+	"sort"
```
Comment (hanfei1991) [inline expression/constant_test.go]: put sys package in top of all packages.

**Comment by lamxTyler** [comment]
@hanfei1991 PTAL

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**